core/workspace_svg.js

```diff
@@ -131,12 +131,20 @@ Blockly.WorkspaceSvg.prototype.resizeHandlerWrapper_ = null;
 
 /**
  * The render status of an SVG workspace.
- * Returns `true` for visible workspaces and `false` for non-visible,
- * or headless, workspaces.
+ * Returns `false` for headless workspaces and true for instances of
+ * `Blockly.WorkspaceSvg`.
  * @type {boolean}
  */
 Blockly.WorkspaceSvg.prototype.rendered = true;
 
+/**
+ * Whether the workspace is visible.  False if the workspace has been hidden
+ * by calling `setVisisble(false)`.
+ * @type {boolean}
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.isVisible = true;
```

Comment:
Make this `@private` `isVisible_`, adding `getVisibility()` to discourage `workspace.isVisible = false;`.

Reply:
done